python_qt_binding/binding_helper.py

```diff
@@ -1,5 +1,3 @@
-#!/usr/bin/env python
```

Comment:
Please keep this line. It allows to invoke the file directly to test the selected binding.

Reply:
Reverting this line